### PR TITLE
Eventbus Debug Widget

### LINKED ISSUE #262: Eventbus debug widget
Add an event bus debug widget for easier connection state inspection and bug fixing.
An implementation could look like:
image
image

### PR DESCRIPTION
**Pull request by fussel178**
### Summary 

Implementation of the eventbus debug widget

### Details 

- **common**: Add the eventbus debug widget. To use it, simply add the `commonWidgets` to the widgets list of the `CommonWrapper` component.
- **template**: Import common widgets into application by default

### Additional information 

https://user-images.githubusercontent.com/52416718/120874130-a6d8f480-c594-11eb-9906-8e6218343c87.mp4

### Related links 

- Fixes #262 

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

File: packages/telestion-client-common/package.json [hunk 1]
```diff
@@ -54,9 +54,11 @@
 		}
 	],
 	"dependencies": {
+		"@react-spectrum/table": "^3.0.0-alpha.12",
```
Comment (pklaschka) [inline packages/telestion-client-common/package.json:57]: As `@adobe/react-spectrum` is a peer dependency, could we please make `@react-spectrum/table` a peer dependency as well, please?

File: packages/telestion-client-common/src/widgets/eventbus-debug-widget/components/eventbus-details.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	const items = useErrorMessageState(lastErrorMessage);
+
+	if (!eventBus) {
+		return (
+			<IllustratedMessage>
+				<Unavailable />
+				<Heading>No EventBus instance</Heading>
+				<Content>
+					There is currently no eventbus instance. Do you forget to create an
+					eventbus connection?
```
Comment (pklaschka) [inline packages/telestion-client-common/src/widgets/eventbus-debug-widget/components/eventbus-details.tsx:55]:
```suggestion
					There is currently no eventbus instance. Did you forget to create an
					eventbus connection?
```

**Review by pklaschka — CHANGES_REQUESTED** [review]
Apart from the two things I suggested inline, it looks good 😃

File: packages/telestion-client-common/package.json (hunk 1 above)
Comment (fussel178) [reply packages/telestion-client-common/package.json]: Done in [`7627af2` (#645)](https://github.com/wuespace/telestion-client/pull/645/commits/7627af2391efa23be3f546192eca17c3a4b6b045).
I also added `@react-spectrum/table` to the `@wuespace/telestion-client-template` package.

**Review by pklaschka — APPROVED** [review]
LGTM! Thank you for your contribution 👍 🎉

**Pull request merged**